.github/workflows/spelling2.yml

```diff
@@ -0,0 +1,27 @@
+# spelling.yml is blocked per https://github.com/check-spelling/check-spelling/security/advisories/GHSA-g86g-chm8-7r2p
```

Comment:
I tried to open the URL, but got a 404 error.

Reply:
I'm waiting to publish the GHSA until after I get the majority of users updated.